drivers/mpu9250/MPU9250.hpp

```diff
@@ -151,7 +151,8 @@ namespace DriverFramework
 #define BITS_ACCEL_CONFIG_16G		0x18
 
 // This is ACCEL_FCHOICE_B which is the inverse of ACCEL_FCHOICE
-#define BITS_ACCEL_CONFIG2_BW_1130HZ	0x08
+#define BITS_ACCEL_CONFIG2_BW_1046HZ	0x08
```

Comment:
Where did you see this value? I see 1.13 K in the [register map](https://cdn.sparkfun.com/assets/learn_tutorials/5/5/0/MPU-9250-Register-Map.pdf) (page 15).